redisstorage.go

```diff
@@ -115,3 +167,30 @@ func (s *Storage) getIDStr(ID uint64) string {
 func (s *Storage) getCookieID(c string) string {
 	return fmt.Sprintf("%s:cookie:%s", s.Prefix, c)
 }
+
+func stringify(cookies []*http.Cookie) string {
+	// Stringify cookies.
+	cs := make([]string, len(cookies))
+	for i, c := range cookies {
+		cs[i] = c.String()
+	}
+	return strings.Join(cs, "\n")
+}
+
+func unstringify(s string) []*http.Cookie {
+	h := http.Header{}
+	for _, c := range strings.Split(s, "\n") {
+		h.Add("Set-Cookie", c)
+	}
+	r := http.Response{Header: h}
+	return r.Cookies()
+}
```

Comment:
The previous version of this was producing malformed cookies (each cookie became multiple garbage cookies) because it was using `http.Request`.

Reply:
eh, stupid bug.. thanks for the fix.